src/docs/sdk/event-payloads/threads.mdx

```diff
@@ -26,13 +26,19 @@ attribute to cross-reference this thread.
 
 `crashed`
 
-: _Optional_. A flag indicating whether the thread crashed. Defaults to `false`.
+: _Optional_. A flag indicating whether the thread was the cause for the event being sent, e.g. due to a crash.
+Defaults to `false`.
 
 `current`
 
-: _Optional_. A flag indicating whether the thread was in the foreground.
+: _Optional_. A flag indicating whether the thread was in the foreground. A thread is in foreground when it's executing.
 Defaults to `false`.
 
+`ui`
+
+: _Optional_. If applicable, a flag indicating whether the thread was responsible for rendering the user interface.
+On mobile platforms this is oftentimes referred to as the "main thread" or "ui thread".
```

Comment:
Should we set a default of `false`, or should the default be `null`?

Reply:
Good point, IMO I would default to `null` in this case, as a `false` default value may is not meaningful on some platforms.